probcal/enums.py

```diff
@@ -42,6 +42,8 @@ class ImageDatasetName(Enum):
     MNIST = "mnist"
     COCO_PEOPLE = "coco_people"
     AAF = "aaf"
+    OOD_COCO_PEOPLE = "ood_coco_people"
```

Comment:
I think this enum value is old and can be dropped. Probably got in on an earlier merge